packages/functions/src/optionSchemas.ts

```diff
@@ -96,6 +96,26 @@ export const optionSchemas: Record<string, CustomFunctionOptionsSchema> = {
       type: `"length" function has invalid options specified. Example valid options: { "min": 2 }, { "max": 5 }, { "min": 0, "max": 10 }`,
     },
   },
+  or: {
+    type: 'object',
+    properties: {
+      properties: {
+        type: 'array',
+        items: {
+          type: 'string',
+        },
+        minItems: 1, // OR is valid with one item (then it is redundant with 'defined' function)
```

Comment:
```suggestion
        minItems: 2,
```
IMO, we should recommend usage of **defined** if only one entry is anticipated